lib/telephony/configuration.rb

```diff
@@ -50,6 +50,8 @@ class Configuration
     attr_writer :adapter
     attr_reader :pinpoint
     attr_accessor :logger
+    attr_accessor :voice_pause_time
+    attr_accessor :voice_rate
```

Comment:
Should we have defaults for these?

Reply:
channelling my inner @pauldoomgov.... no, because hardcoded defaults end up being bad

Reply:
Fair enough 👍